frontend/src/components/pages/HomePage/SearchAndFilters.tsx

```diff
@@ -103,6 +76,43 @@ const SearchAndFilters = ({
   setBuilding,
   setFlagged,
 }: Props): React.ReactElement => {
+  const [userLabels, setUserLabels] = useState<UserLabel[]>();
+  const [residentLabels, setResidentLabels] = useState<ResidentLabel[]>();
+
+  const getUsers = async () => {
+    const data = await UserAPIClient.getUsers({ returnAll: true });
+    const users = data?.users;
+    if (users) {
+      const labels = users.map((user: User) => {
+        return {
+          id: user.id,
+          label: `${user.firstName} ${user.lastName}`,
+          value: user.id,
+        } as UserLabel;
+      });
+      setUserLabels(labels);
+    } else {
+      setUserLabels(undefined);
```

Comment:
Don't need to include this else clause, this is only called when the component renders and the state is already undefined above on line 79